src/App.js

```diff
@@ -1,31 +1,34 @@
-import React, {Component} from 'react';
-import Menu from './menu';
-import data from './data.json';
-
+import React, { Component } from "react";
+import Menu from "./menu";
+import data from "./data.json";
 
 class App extends Component {
-
   constructor(props) {
-    super(props)
-    this.state = { selectedMeal: "" }
-
-  }
-
-  handleChange = (evt) => {
-    this.setState({selectedMeal: evt.target.value})
+    super(props);
+    this.state = { selectedMeal: "salmonLinguine" };
   }
 
-  }
+  handleChange = evt => {
+    this.setState({ selectedMeal: evt.target.value });
+  };
 
   render() {
-  //  const recip = data.recipes.find((recipeObj) => )
+    const selectedIngredients = data.recipes.find(obj => {
+      return obj.id === this.state.selectedMeal;
+    }).ingredients;
+
+    const listItems = selectedIngredients.map(ingredient => {
+      return <p>{ingredient}</p>;
```

Comment:
I would use [li](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li) since it's a list. Ingredients is an unordered list.